src/test/java/uk/gov/pay/card/model/CardTypeTest.java

```diff
@@ -19,15 +15,25 @@ public void shouldFindCardClass() {
 
     @Test
     public void shouldThrowNullPointerException_WhenNullValuePassed_forCardClass() {
-        thrown.expect(java.lang.NullPointerException.class);
-        thrown.expectMessage("Value cannot be null for paymentGatewayRepresentation");
-        new CardInformation("A", null, "A", 1L, 2L);
+        try {
+            new CardInformation("A", null, "A", 1L, 2L);
+            fail("The constructor was expected to throw an exception");
+        } catch (NullPointerException e) {
+            assertEquals("Value cannot be null for paymentGatewayRepresentation", e.getMessage());
+        } catch (Throwable e) {
+            fail("An unexpected exception was thrown by the constructor: " + e);
+        }
```

Comment:
Not tried it, but apparently you can actually do something like this:

```java
var thrown = assertThrows(NullPointerException.class, () -> new CardInformation("A", null, "A", 1L, 2L));
assertEquals("Value cannot be null for paymentGatewayRepresentation", thrown.getMessage());
```

Reply:
Cunning